### PR TITLE
Usage doc update

### PR DESCRIPTION
**Pull request by tzumainn**

### CODE REVIEW

File: docs/source/usage/common_scenarios.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+================
+
+Lessee Node Assignment
+----------------------
+
+Lessee node assignment can be performed by the administrator or node owner. It can be executed using the Ironic CLI:
+
+  .. prompt:: bash $
+
+    openstack baremetal node set –lessee <lessee project id> <node>
```
Comment (DanNiESh) [inline docs/source/usage/common_scenarios.rst:11]: Isn't it supposed to be --lessee?

File: docs/source/usage/common_scenarios.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+However, if you require reporting on historical usage, it is recommended that you use the `ESI Leap`_ service to create a lease that can be tracked and reported upon. This can be done as follows:
+
+  .. prompt:: bash $
+
+    openstack esi lease create –start-time <optional start time> –end-time <optional end time> <node> <lessee project id>
```
Comment (DanNiESh) [inline docs/source/usage/common_scenarios.rst:17]: --start-time and --end-time

File: docs/source/usage/common_scenarios.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+An administrator or node owner can also use ESI-Leap to offer up a node for any project to claim:
+
+  .. prompt:: bash $
+
+    openstack esi offer create –start-time <optional start time> –end-time <optional end time> <node>
```
Comment (DanNiESh) [inline docs/source/usage/common_scenarios.rst:23]: Same as above.

File: docs/source/usage/common_scenarios.rst [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+A lessee can then claim the node, creating a lease:
+
+  .. prompt:: bash $
+
+    openstack esi offer list           # note UUID of offer to claim
+    openstack esi offer claim –start-time <optional start time> –end-time <optional end time> <offer UUID>
```
Comment (DanNiESh) [inline docs/source/usage/common_scenarios.rst:30]: --start-time and --end-time

File: docs/source/usage/common_scenarios.rst [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  .. prompt:: bash $
+
+    openstack network create <network name>
+    openstack subnet create --subnet-range 192.168.17.0/24 --allocation-pool start=192.168.17.10,end=192.168.17.250 --network <network name> <subnet name>
+
+The created network will automatically have an assigned VLAN. This network can then be attached to a node as follows:
+
+  .. prompt:: bash $
+
+    openstack esi node network attach –network <network name> <node>
```
Comment (DanNiESh) [inline docs/source/usage/common_scenarios.rst:59]: --network

File: docs/source/usage/common_scenarios.rst [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+In order for the lessee to access a node on a private network, they can do one of two things:
+
+* Contact the ESI administrator to connect the VLAN to an accessible node or VM outside of ESI
+* Use a trunk port whose native network is a VLAN that the lessee can already access - usually a public or external network - and which has a tagged network corresponding to the private network:
+
+  .. prompt:: bash $
+
+    openstack esi trunk create --native-network <accessible network> <trunk name>
+    openstack esi trunk add network –tagged-networks <private network> <trunk name>
+    openstack esi node network attach –port <trunk port> <node>
```
Comment (DanNiESh) [inline docs/source/usage/common_scenarios.rst:70]: --port and --tagged-networks

File: docs/source/usage/common_scenarios.rst (hunk 1 above)
Comment (tzumainn) [reply docs/source/usage/common_scenarios.rst]: Whoops - I think copying and pasting from the google doc messed things up. Fixed!

File: docs/source/usage/common_scenarios.rst (hunk 2 above)
Comment (tzumainn) [reply docs/source/usage/common_scenarios.rst:17]: Fixed!

File: docs/source/usage/common_scenarios.rst (hunk 3 above)
Comment (tzumainn) [reply docs/source/usage/common_scenarios.rst]: Fixed!

File: docs/source/usage/common_scenarios.rst (hunk 4 above)
Comment (tzumainn) [reply docs/source/usage/common_scenarios.rst]: Fixed!

File: docs/source/usage/common_scenarios.rst (hunk 5 above)
Comment (tzumainn) [reply docs/source/usage/common_scenarios.rst]: Fixed!

File: docs/source/usage/common_scenarios.rst (hunk 6 above)
Comment (tzumainn) [reply docs/source/usage/common_scenarios.rst]: Fixed!

**Comment by tzumainn** [comment]
Thanks for catching the "--" issue! It should be fixed now.

**Comment by rd4398** [comment]
Looks good to me!

**Review by DanNiESh — APPROVED** [review]
LGTM!

**Pull request merged**